src/AspNetCoreModuleV2/RequestHandler/dllmain.cxx

```diff
@@ -21,7 +21,8 @@ IHttpServer *       g_pHttpServer = NULL;
 HINSTANCE           g_hWinHttpModule;
 HINSTANCE           g_hAspNetCoreModule;
 HANDLE              g_hEventLog = NULL;
-
+PCSTR               g_szDebugLabel = "ASPNET_CORE_MODULE_REQUEST_HANDLER";
+std::map<std::wstring, std::wstring> requestHandlerConfigurationMap;
```

Comment:
Don't use the dictionary, compare name and stick value into a variable.

Reply:
😢